src/styles.css

```
@@ -121,6 +121,55 @@ body.modal-open{overflow:hidden;  height:100vh;}
+  z-index:9999;
+
+  opacity:0;
+  visibility:hidden;
+
+  transform:translateY(6px);
+  transition:
+    opacity .18s ease,
+    transform .18s ease,
+    visibility .18s;
+
+  box-shadow:0 10px 25px rgba(0,0,0,.18);
```

Stylelint (17.11.1)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)